src/components/FeedbackForm/index.js

```diff
@@ -51,12 +52,21 @@ export const FeedbackForm = () => {
             if (res.status === 201) {
               setInputVal({ value: '', errorMsg: '' });
               setDropDownValue({ ...dropDownValue, value: '' });
+              setDoneMsg({ value: 'Msg is Sended', status: true });
+              setInterval(() => {
+                setDoneMsg({ value: '', status: false });
+              }, 3000);
             }
           })
           .catch(err => {
-            console.log(err.status);
-            setInputVal({ value: '', errorMsg: '' });
-            setDropDownValue({ ...dropDownValue, value: '' });
+            console.log(err);
+            setDoneMsg({
+              value: 'you can send only 5 feedback in day',
+              status: false
+            });
+            setInterval(() => {
+              setDoneMsg({ value: '', status: false });
+            }, 3000);
```

Comment:
why you used setInterval here!

Reply:
to make msg is empty after 3 seconds and make div hidden

Reply:
1. if this is a requirement, then it should be `setTimeout` not `setInterval`

2. this is not a requirement so don't implement it.